Add MongoDB description (#5519)
redwoodjs/redwood: docs/docs/tutorial/chapter2/getting-dynamic.md

First let's define the data structure for a post in the database. Open up `api/db/schema.prisma` and add the definition of our Post table (remove any "sample" models that are present in the file, like the `UserExample` model). Once you're done, the entire schema file should look like:

<Tabs groupId="sql-mongo">
<TabItem value="sql" label="SQLite">

> ```Javascript title="api/db/schema.prisma"

Reviewer: It looks like docusaurus doesn't recognize "Javascript". Lowercasing it again should bring back syntax highlighting
```suggestion
```javascript title="api/db/schema.prisma"
```

// highlight-end
```

</TabItem>
<TabItem value="mongo" label="MongoDB">

> ```Javascript

Reviewer: It looks like docusaurus doesn't recognize "Javascript". Lowercasing it again should bring back syntax highlighting
```suggestion
```javascript
```

Integers make for nicer URLs like https://redwoodblog.com/posts/123 instead of https://redwoodblog.com/posts/eebb026c-b661-42fe-93bf-f1a373421a13.

> The MongoDB is using strings as `id` which is mapping to the mongo built-in object id(`_id`)

Reviewer: Tried to improve the copy a bit here. Is this still accurate?
```suggestion
The MongoDB example uses `String` since it maps to MongoDB's built-in object id (`_id`)
```

yarn rw prisma migrate dev
```

:::warning

> If you are using a MongoDB database you shall not run any migrations as this is not possible with a document database as MongoDB. You want to run `npx prisma generate` instead at this point. To apply the schema to your database you can run `npx prisma db push` which will make collections of your models.

Reviewer: Just tried to improve the readability a bit here:
```suggestion
If you're using MongoDB, you shouldn't run any migrations since they're not possible with document databases such as MongoDB. You'll want to run `yarn rw prisma generate` at this point instead. To apply the schema to your database, you can run `yarn rw prisma db push`, which will make collections of your models.
```
